front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputDetails/index.tsx

```diff
@@ -191,19 +193,21 @@ const InputDetails = ({
   return (
     <>
       <Container data-testid="insightsInputDetails">
-        <CategoryList>
-          {previewedInput.relationships?.suggested_categories.data.map(
-            (category) => (
-              <Category
-                id={category.id}
-                key={category.id}
-                inputId={previewedInput.id}
-                variant="suggested"
-                size="large"
-              />
-            )
-          )}
-        </CategoryList>
+        {nlpFeatureFlag && (
```

Comment:
Ah nice, it's not blocking but we usually aren't that careful with flagging, mostly we prevent creation of the protected resources but always show them. Makes more sense like this, though, especially is we process some things automatically...

Reply:
Yes, this was my thinking as well!